CcWorks/Workers/ReviewWorker.cs

```diff
@@ -82,34 +123,76 @@ await issue.WorkflowTransitionAsync("Submit for Review", new WorkflowTransitionU
 
                 var currentUserIdJson = await GithubHelper.Query("query {viewer {id}}", commonSettings.GithubToken);
                 var userId = currentUserIdJson["viewer"]["id"].Value<string>();
-
                 var nodes = jPullRequest["assignees"]["nodes"] as JArray ?? new JArray();
-                foreach (var node in nodes)
+
+                if (nodes.Any(node => node["id"].Value<string>() == userId))
                 {
-                    if (node["id"].Value<string>() == userId)
-                    {
-                        Console.WriteLine("already assigned");
-                        return;
-                    }
+                    Console.WriteLine("already assigned");
+                }
+                else
+                {
+                    var assignPrMutation = string.Format(MutationAddAssigneePRFormat, jPullRequest["id"].Value<string>(), userId);
+                    await GithubHelper.Query(assignPrMutation, commonSettings.GithubToken);
+                    Console.WriteLine("done");
                 }
 
-                var prId = jPullRequest["id"].Value<string>();
-                var assignPrMutation = @"mutation
-                    {
-                      addAssigneesToAssignable(input: {
-                        assignableId: """ + prId + @""", 
-                        assigneeIds: [""" + userId + @"""]}) { clientMutationId }}";
+                if (settings.AssignReviewLabel)
```

Comment:
In your code `AssignReviewLabel` depends on `AssignPr`. So if user will set up tool like:
```json
{
    "assignPr": false,
    "assignReviewLabel": true
}
```
then label won't be added.
I suggest to make these options independent.